x11-packages/pcsx-rearmed/build.sh

```diff
@@ -23,20 +23,19 @@ termux_pkg_auto_update() {
 }
 
 termux_step_pre_configure() {
-	CFLAGS+=" $CPPFLAGS"
-	if [ "$TERMUX_ARCH" = "arm" ]; then
-		termux_setup_no_integrated_as
-	fi
+	CFLAGS+=' '"$CPPFLAGS"
+	CFLAGS="${CFLAGS/-Oz/-Os}"
+	[ "${TERMUX_ARCH}" = 'arm' ] && termux_setup_no_integrated_as
 }
 
 termux_step_configure() {
-	sh ./configure
+	bash ./configure
 }
 
 termux_step_make_install() {
-	install -Dm755 pcsx $TERMUX_PREFIX/bin/pcsx
-	mkdir -p $TERMUX_PREFIX/etc/pcsx $TERMUX_PREFIX/lib/pcsx_plugins
-	cp -r frontend/pandora/skin $TERMUX_PREFIX/etc/pcsx/
-	install -m755 plugins/*.so $TERMUX_PREFIX/lib/pcsx_plugins/
-	ln -fs ../../lib/pcsx_plugins $TERMUX_PREFIX/etc/pcsx/plugins
+	install -D -m755 pcsx ${TERMUX_PREFIX}/bin/pcsx
+	mkdir -p ${TERMUX_PREFIX}/etc/pcsx ${TERMUX_PREFIX}/lib/pcsx_plugins
+	cp -f -r frontend/pandora/skin ${TERMUX_PREFIX}/etc/pcsx/
+	install -m755 plugins/*.so ${TERMUX_PREFIX}/lib/pcsx_plugins/
+	ln -f -s ../../lib/pcsx_plugins ${TERMUX_PREFIX}/etc/pcsx/plugins
```

Comment:
Love the braces personally.
But again, this doesn't fix anything.
You just unclumped the flags and added braces.